### ISSUE TITLE
when I use same parameters to train mot17_half,I get different results every time.

### ISSUE THREAD

**Issue by monsters-s** [xingyizhou/CenterTrack#125]
I have tried to add these codes  as user 'databatman' in (https://github.com/pytorch/pytorch/issues/7068),in main.py between 30th lines and 31 lines:
random.seed(seed)
    os.environ['PYTHONHASHSEED'] = str(seed)
    np.random.seed(seed)
    torch.manual_seed(seed)
    torch.cuda.manual_seed(seed)
    torch.cuda.manual_seed_all(seed)  # if you are using multi-GPU.
    torch.backends.cudnn.benchmark = False
    torch.backends.cudnn.deterministic = True

But I still can't get same results. 
Is there a random seed I haven't noticed?
